src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inverseObject = {};
+
+  for (const key in robot) {
+    const ROBOT_KEY = robot[key];
```

Comment:
it's better to rename this variable to lowercase, basically we use upper + snake keys for static variables that won't be changed

learn more about javascript naming convention - [link](https://www.robinwieruch.de/javascript-naming-conventions/)